terraform/helm/autoscaling/Chart.yaml

```diff
@@ -4,5 +4,5 @@ version: 0.1.0
 
 dependencies:
   - name: metrics-server
-    version: 3.8.2
+    version: 3.10.0
```

Comment:
you'll need to update the cached version of the chart in `charts/` too

Reply:
Done.